camera/EmulatedCameraDevice.h

```diff
@@ -393,11 +394,7 @@ class EmulatedCameraDevice {
              * Return:
              *  NO_ERROR on success, or an appropriate error status.
              */
-            inline status_t startThread(bool one_burst)
-            {
-                mOneBurst = one_burst;
-                return run(NULL, ANDROID_PRIORITY_URGENT_DISPLAY, 0);
```

Comment:
This looks like a much simpler version of this function, compared to https://github.com/mozilla-b2g/android-development/commit/972ad0800dc7592d029ec0f3d2155ed020138975#diff-9755028951815e61d8c66da8d446f20bL356

I'm happy to see that, but just want to confirm that nothing is missing here.